apps/vmq_server/src/vmq_mqtt_fsm.erl

```diff
@@ -108,7 +108,8 @@ init(Peer, Opts) ->
                     },
     TraceFun = vmq_config:get_env(trace_fun, undefined),
 
-    TRef = vmq_mqtt_fsm_util:send_after(?CLOSE_AFTER, close_timeout),
+    ConnectTimeout = vmq_config:get_env(connect_timeout, 5000),
+    TRef = vmq_mqtt_fsm_util:send_after(ConnectTimeout, close_timeout),
```

Comment:
do you still need the `CLOSE_AFTER` macro? I think it can be removed to leave the code clean

Reply:
removed!